#Classes.msg
string[] classes
bool use_prompts

# Type invariant: prompts and groups are represented as json-serialized strings of dictionaries
string[] prompts
string[] groups